tests/__utils.ts

```diff
@@ -51,13 +51,15 @@ export async function generateSchemaAndTypes(
   types: any[],
   options?: TransformOptions & {
     experimentalCRUD?: boolean
-    plugins?: Nexus.core.NexusPlugin[]
+    plugins?: Nexus.core.NexusPlugin[],
+    scalars?: Record<string, GQL.GraphQLScalarType>
```

Comment:
Shouldn't this be not an unbounded indexed type but an object for scalars mapping to the set of support prisma scalars? e.g. `{ json: ..., date: ... }`

Reply:
Done in c3be2a5 and c9c2a86